Templates/Databases/MicrosoftSQL/CSV file to MS SQL Database insert/process.json

```diff
@@ -0,0 +1,77 @@
+﻿{
+	"ProcessTemplates": [
+		{
+			"Name": "CSV file to MS SQL Database insert",
+			"Modified": "2024-04-30T07:59:27.8783994",
+			"Modifier": "david.roberts@frends.com",
+			"Tags": [
+				"MS SQL Server",
+				"CSV"
+			],
+			"Description": "This Process will take the contents of a CSV file. CSV is converted to JSON before creating and executing INSERT statement to MS SQL database.",
```

Comment:
When you use batch operation this description needs a little update in terms of how things operate.